app/controllers/users_controller.rb

```diff
@@ -5,5 +5,6 @@ def index
 
   def show
     @user = User.find(params[:id])
+    @posts = @user.posts
```

Comment:
- [ ] You should make sure that the N+1 problem is solved when fetching all posts and their comments for a user.
N+1 problem still persists 
![Screenshot from 2021-11-06 04-03-46](https://user-images.githubusercontent.com/1642014/140592803-17c3043c-ffc0-4c66-88b1-05e1dce1c726.png)

Tip: You can use the bullet gem to detect N+1 Problems

Reply:
Thank you!